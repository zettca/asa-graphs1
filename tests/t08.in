70 300
34
34 59
55 48
55 37
55 35
55 18
55 57
55 41
55 25
55 31
50 67
50 21
50 41
50 25
61 60
61 37
61 68
61 29
61 24
61 66
61 15
61 28
61 10
61 19
61 14
61 63
61 22
61 31
61 46
9 60
9 48
9 37
9 69
9 2
9 16
9 29
9 36
9 66
9 15
9 33
9 4
9 49
9 63
9 46
40 59
40 51
53 65
53 7
53 17
53 45
65 56
65 70
67 6
67 42
67 41
67 8
67 20
60 68
60 69
60 16
60 43
60 57
60 27
60 33
60 44
60 49
60 19
60 14
60 22
60 46
60 52
60 23
54 69
54 2
54 33
54 10
54 13
54 14
54 1
54 62
54 63
54 22
48 26
48 24
48 35
48 18
48 57
48 44
48 28
48 19
48 46
48 52
30 51
30 13
30 62
3 6
3 26
3 18
3 27
3 66
3 15
3 12
3 28
3 19
3 52
3 8
7 17
7 11
7 56
7 38
7 64
17 56
17 64
17 58
6 18
6 57
6 41
6 49
6 8
37 26
37 24
37 35
37 18
37 66
37 46
37 8
37 23
37 20
68 29
68 43
68 36
68 57
68 15
68 44
68 49
68 28
68 10
68 63
68 46
68 52
68 23
26 36
26 27
26 15
26 12
26 25
69 2
69 16
69 4
69 1
69 23
2 16
2 24
2 43
2 66
2 46
16 36
16 57
16 66
16 15
16 33
16 4
16 10
16 19
16 31
16 52
51 32
21 47
42 35
42 36
42 18
42 57
42 19
42 31
42 8
42 20
11 56
11 45
11 64
11 5
29 18
29 4
29 12
29 28
29 46
29 23
56 5
24 43
24 36
24 49
24 10
24 14
24 46
24 52
38 70
38 5
35 36
35 41
35 12
35 49
35 19
35 8
35 20
43 66
43 4
43 44
43 28
43 10
43 14
43 1
43 63
43 22
43 23
45 64
45 58
47 39
36 27
36 15
36 28
36 46
36 8
36 23
18 12
18 28
18 19
18 31
18 46
18 23
18 20
64 58
64 5
57 27
57 66
57 4
57 12
57 28
57 46
57 52
57 8
27 66
27 44
27 28
27 19
27 46
27 8
27 20
32 62
66 44
66 49
66 63
66 31
66 52
15 28
15 19
15 31
15 8
33 4
33 44
33 28
33 63
33 46
33 52
41 12
41 25
41 8
4 44
4 49
4 10
4 19
4 63
4 22
12 19
12 25
12 31
44 49
44 63
44 31
44 46
39 58
49 52
49 20
28 19
28 14
28 22
10 14
10 1
10 63
10 23
13 1
19 46
19 52
14 63
14 52
14 23
25 8
1 22
22 46
22 23
46 23
52 23
8 20
