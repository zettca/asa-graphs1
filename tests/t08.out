20
1
1
1
2
2
2
8
13
12
8
3
1
1
1
1
1
3
5
2
1
